gapic/utils/lines.py

```diff
@@ -40,6 +40,47 @@ def sort_lines(text: str, dedupe: bool = True) -> str:
     return f'{leading}{answer}{trailing}'
 
 
+def get_subsequent_line_indentation_level(list_item: str) -> int:
+    """
+    Given a list item return the indentation level for subsequent lines.
+    For example, if it is a numbered list, the indentation level should be 3
+    as shown below.
+
+    Here subsequent lines should be indented by 2
+
+    - The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 2
+
+    + The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 3
+
+    1. The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+       over the lazy dog
+
+    """
+    if len(list_item) >= 2 and list_item[0:2] in ['- ', '+ ']:
+        indentation_level = 2
+    elif len(list_item) >= 3 and list_item[0].isdigit() and list_item[1:3] == '. ':
+        indentation_level = 3
+    else:
+        # Don't use any intentation level if the list item marker is not known
+        indentation_level = 0
+    return indentation_level
+
+
+def is_list_item(list_item: str) -> bool:
+    """
+    Given a string return a boolean indicating whether a list is identified.
+    """
+    if len(list_item) < 3:
+        return False
+    return list_item.startswith('- ') or list_item.startswith('+ ') or (list_item[0].isdigit() and list_item[1:].startswith('. '))
```

Comment:
Again, issue with double-digits.

```suggestion
    return list_item.startswith('- ') or list_item.startswith('+ ') or re.match("^\d+\. ", list_item)
```